### PR TITLE
fix(v4): update swizzle based on v4

### PR DESCRIPTION
**Pull request by salihozdemir**
Updated swizzle `transform` functions based on v4.

- [x] All swizzleable item tested

### CODE REVIEW

File: packages/antd/refine.config.js [hunk 1]
```diff
@@ -288,23 +288,8 @@ module.exports = {
...
-                    return (
-                        <Refine
-                            catchAll={ErrorPage}
-                            /* ... */
-                        />
-                    );
-                }
-                \`\`\`
+                **\`Info:\`**
+                If you want to see an examples of error page in use, you can refer to the documentation at https://refine.dev/docs/packages/documentation/routers/react-router-v6
```
Comment (aliemir) [inline packages/antd/refine.config.js:292]:
```suggestion
                If you want to see an examples of error page in use, you can refer to the documentation at https://refine.dev/docs/packages/documentation/routers
```

File: packages/chakra-ui/refine.config.js [hunk 2]
```diff
@@ -233,23 +233,8 @@ module.exports = {
...
-                    return (
-                        <Refine
-                            catchAll={ErrorPage}
-                            /* ... */
-                        />
-                    );
-                }
-                \`\`\`
+                **\`Info:\`**
+                If you want to see an examples of error page in use, you can refer to the documentation at https://refine.dev/docs/packages/documentation/routers/react-router-v6
```
Comment (aliemir) [inline packages/chakra-ui/refine.config.js:237]:
```suggestion
                If you want to see an examples of error page in use, you can refer to the documentation at https://refine.dev/docs/packages/documentation/routers
```

File: packages/chakra-ui/refine.config.js [hunk 3]
```diff
@@ -276,23 +261,8 @@ module.exports = {
...
-                    return (
-                        <Refine
-                            LoginPage={AuthPage}
-                            /* ... */
-                        />
-                    );
-                }
-                \`\`\`
+                **\`Info:\`**
+                If you want to see an example of authentication pages in use, you can refer to the documentation at https://refine.dev/docs/packages/documentation/routers/react-router-v6/#how-to-handle-optional-authentication-redirects-and-layouts-with-authentication
```
Comment (aliemir) [inline packages/chakra-ui/refine.config.js:265]:
```suggestion
                If you want to see an example of authentication pages in use, you can refer to the documentation at https://refine.dev/docs/packages/documentation/routers
```

File: packages/core/refine.config.js [hunk 4]
```diff
@@ -9,23 +9,8 @@ module.exports = {
...
-                    return (
-                        <Refine
-                            catchAll={ErrorPage}
-                            /* ... */
-                        />
-                    );
-                }
-                \`\`\`
+                **\`Info:\`**
+                If you want to see an examples of error page in use, you can refer to the documentation at https://refine.dev/docs/packages/documentation/routers/react-router-v6
```
Comment (aliemir) [inline packages/core/refine.config.js:13]:
```suggestion
                If you want to see an examples of error page in use, you can refer to the documentation at https://refine.dev/docs/packages/documentation/routers
```

File: packages/core/refine.config.js [hunk 5]
```diff
@@ -38,23 +23,8 @@ module.exports = {
...
-                    return (
-                        <Refine
-                            LoginPage={AuthPage}
-                            /* ... */
-                        />
-                    );
-                }
-                \`\`\`
+                **\`Info:\`**
+                If you want to see an example of authentication pages in use, you can refer to the documentation at https://refine.dev/docs/packages/documentation/routers/react-router-v6/#how-to-handle-optional-authentication-redirects-and-layouts-with-authentication
```
Comment (aliemir) [inline packages/core/refine.config.js:27]:
```suggestion
                If you want to see an example of authentication pages in use, you can refer to the documentation at https://refine.dev/docs/packages/documentation/routers
```

File: packages/mantine/refine.config.js [hunk 6]
```diff
@@ -276,23 +261,8 @@ module.exports = {
...
-                    return (
-                        <Refine
-                            LoginPage={AuthPage}
-                            /* ... */
-                        />
-                    );
-                }
-                \`\`\`
+                **\`Info:\`**
+                If you want to see an example of authentication pages in use, you can refer to the documentation at https://refine.dev/docs/packages/documentation/routers/react-router-v6/#how-to-handle-optional-authentication-redirects-and-layouts-with-authentication
```
Comment (aliemir) [inline packages/mantine/refine.config.js:265]:
```suggestion
                If you want to see an example of authentication pages in use, you can refer to the documentation at https://refine.dev/docs/packages/documentation/routers
```

File: packages/mui/refine.config.js [hunk 7]
```diff
@@ -276,23 +261,8 @@ module.exports = {
...
-                    return (
-                        <Refine
-                            LoginPage={AuthPage}
-                            /* ... */
-                        />
-                    );
-                }
-                \`\`\`
+                **\`Info:\`**
+                If you want to see an example of authentication pages in use, you can refer to the documentation at https://refine.dev/docs/packages/documentation/routers/react-router-v6/#how-to-handle-optional-authentication-redirects-and-layouts-with-authentication
```
Comment (aliemir) [inline packages/mui/refine.config.js:265]:
```suggestion
                If you want to see an example of authentication pages in use, you can refer to the documentation at https://refine.dev/docs/packages/documentation/routers
```

**Review by aliemir — CHANGES_REQUESTED** [review]
Hey @salihozdemir, requested some changes about the docs URLs. We've added an index page at #3689. We can use it as the entry point for the docs about router providers.

**Review by aliemir — DISMISSED** [review]

File: packages/antd/refine.config.js [hunk 8]
```diff
@@ -288,23 +288,8 @@ module.exports = {
...
-                    return (
-                        <Refine
-                            catchAll={ErrorPage}
-                            /* ... */
-                        />
-                    );
-                }
-                \`\`\`
+                **\`Info:\`**
+                If you want to see an examples of error page in use, you can refer to the documentation at https://refine.dev/docs/packages/documentation/routers
```
Comment (BatuhanW) [inline packages/antd/refine.config.js:292]:
```suggestion
                If you want to see an example of error page in use, you can refer to the documentation at https://refine.dev/docs/packages/documentation/routers
```

File: packages/antd/refine.config.js [hunk 9]
```diff
@@ -331,23 +316,8 @@ module.exports = {
...
-                    return (
-                        <Refine
-                            LoginPage={AuthPage}
-                            /* ... */
-                        />
-                    );
-                }
-                \`\`\`
+                **\`Info:\`**
+                If you want to see an example of authentication pages in use, you can refer to the documentation at https://refine.dev/docs/packages/documentation/routers
```
Comment (BatuhanW) [inline packages/antd/refine.config.js:320]:
```suggestion
                If you want to see examples of authentication pages in use, you can refer to the documentation at https://refine.dev/docs/packages/documentation/routers
```

File: packages/antd/refine.config.js [hunk 10]
```diff
@@ -498,18 +468,24 @@ module.exports = {
                 message: `
                 **\`Warning:\`**
                 If you want to change the default layout;
-                You should pass \`layout/index.tsx\` with the **Layout** prop to the **<Refine/>** component.
+                You should pass the layout components to the **<Layout/>** component.
```
Comment (BatuhanW) [inline packages/antd/refine.config.js:471]:
```suggestion
                You should pass layout related components to the **<Layout/>** component's props.
```

File: packages/antd/refine.config.js (hunk 8 above)
Comment (BatuhanW) [reply packages/antd/refine.config.js:292]: Let's update all references with the same sentence.

File: packages/antd/refine.config.js (hunk 9 above)
Comment (BatuhanW) [reply packages/antd/refine.config.js:320]: Let's update all references with the same sentence.

**Review by BatuhanW — CHANGES_REQUESTED** [review]

**Review by BatuhanW — APPROVED** [review]

**Pull request merged**